clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/CommitEvent.java

```diff
@@ -30,12 +29,7 @@ public abstract class CommitEvent extends CompletableApplicationEvent<Void> {
      */
     private final Map<TopicPartition, OffsetAndMetadata> offsets;
 
-    protected CommitEvent(final Type type, final Map<TopicPartition, OffsetAndMetadata> offsets, final Timer timer) {
-        super(type, timer);
-        this.offsets = validate(offsets);
-    }
-
-    protected CommitEvent(final Type type, final Map<TopicPartition, OffsetAndMetadata> offsets, final long deadlineMs) {
+    public CommitEvent(final Type type, Map<TopicPartition, OffsetAndMetadata> offsets, final long deadlineMs) {
```

Comment:
Is there a reason for loosing the final on the offsets map?

Reply:
ditto: we probably don't need the final but it would be good to be consistent.

Reply:
Added back `final` and changed back to `protected`. Not sure how/why I changed those 🤔